vrk-kpa/suomidigi
# SUOPA-263 Webforms

To test: 
- `make fresh`
- Log in and go to: http://suomidigi.fi.docker.amazee.io/form/tuki-asiakaslahtoisyys
   - Check that the form exists and the theme doesn't break in mobile width
- Go to: http://suomidigi.fi.docker.amazee.io/form/tuki-muutosjohtaminen
   - Check that the form exists and the theme doesn't break in mobile width
